packages/anoncreds/src/repository/AnonCredsCredentialRecord.ts

```diff
@@ -12,24 +13,28 @@ export interface AnonCredsCredentialRecordProps {
   schemaVersion: string
   schemaIssuerId: string
   issuerId: string
+  
 }
 
 export type DefaultAnonCredsCredentialTags = {
   credentialId: string
   linkSecretId: string
   credentialDefinitionId: string
   credentialRevocationId?: string
-  revocationRegistryId?: string
   schemaId: string
   attributes: string[]
+
+    // the following keys can be used for every `attribute name` in credential.
+    [key: `attr::${string}::marker`]: true | undefined
+    [key: `attr::${string}::value`]: string | undefined
 }
 
 export type CustomAnonCredsCredentialTags = {
   schemaName: string
   schemaVersion: string
   schemaIssuerId: string
   issuerId: string
-}
+} & TagsBase
```

Comment:
This can be deleted again

```suggestion
}
```